ip-handling/draft-ietf-rtcweb-ip-handling.xml

```diff
@@ -209,22 +212,33 @@ ipr="trust200902">
           path, and is ideal when media performance is the highest priority, but
           it discloses the most information.</t>
 
-          <t>Default route + associated local addresses: WebRTC MUST follow the
-          kernel routing table rules, which will typically cause media packets to
-          take the same route as the application's HTTP traffic. If an
-          enterprise TURN server is present, the preferred route MUST
-          be through this TURN server. Once an interface has been chosen,
-          the private IPv4 and IPv6 addresses associated with this
-          interface MUST be discovered and provided to the
-          application. This ensures that direct connections can still be
-          established in this mode.</t>
+          <t>Default route + privacy-preserving local addresses: WebRTC MUST
+          follow the kernel routing table rules, which will typically cause
+          media packets to take the same route as the application's HTTP
+          traffic. If an enterprise TURN server is present, the preferred
+          route MUST be through this TURN server. Once an interface has been
+          chosen:
+            <list style="symbols">
+              <t>If the interface has an associated <xref target="RFC4941" />
+              IPv6 address, it MUST be provided to the application.</t>
+              <t>If the interface has an associated <xref target="RFC1918" />
+              IPv4 address, and multicast DNS <xref target="RFC6762" /> is
```

Comment:
Nit: 100.64 is also private IP address space (RFC 6598) - only supposed to be used for CGN installs, but I've seen it used for hosts too.

Reply:
If it's not intended for hosts, perhaps treating it as public is acceptable?